hyperspy/tests/signal/test_1D_tools.py

```diff
@@ -195,17 +195,21 @@ def test_two_spectra(self):
 
     def test_delta_int(self):
         s = self.s[0]
+        s.change_dtype('float')
         s.data[12] *= 10
         s.interpolate_in_between(8, 12, delta=2, kind='cubic')
-        nose.tools.assert_true(
-            np.all(s.data[8: 12] == np.array([44, 95, 139, 155])))
+        print s.data[8:12]
+        np.testing.assert_allclose(
+            s.data[8:12], np.array([44., 95.4, 139.6, 155.]))
 
     def test_delta_float(self):
         s = self.s[0]
-        s.data[12] *= 10
+        s.change_dtype('float')
+        s.data[12] *= 10.
         s.interpolate_in_between(8, 12, delta=0.31, kind='cubic')
-        nose.tools.assert_true(
-            np.all(s.data[8: 12] == np.array([45, 104, 155, 170])))
+        print s.data[8:12]
```

Comment:
Only gets printed if the test fails. And in that case you can see what's the data that did not match